operator-framework/src/test/java/io/javaoperatorsdk/operator/PrimaryToSecondaryMissingIT.java

```diff
@@ -0,0 +1,51 @@
+package io.javaoperatorsdk.operator;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.javaoperatorsdk.operator.junit.LocallyRunOperatorExtension;
+import io.javaoperatorsdk.operator.sample.primarytosecondary.Cluster;
+import io.javaoperatorsdk.operator.sample.primarytosecondary.JobReconciler;
+
+import static io.javaoperatorsdk.operator.PrimaryToSecondaryIT.cluster;
+import static io.javaoperatorsdk.operator.PrimaryToSecondaryIT.job;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+class PrimaryToSecondaryMissingIT {
```

Comment:
Maybe add some description that this unit test is to showcase the role of PtoS mapper and that it it not meant to be used as reusable code.

Reply:
done